src/routes/StackView/StackView.scss

```diff
@@ -9,9 +9,16 @@ $stack-view__parent-note-height: 22%;
   box-shadow: inset 0 0 0 $column__border-width var(--accent-color);
   background: rgba(var(--accent-color-rgb), 0.42);
 }
+[theme="dark"] {
+  .stack-view__portal {
+    backdrop-filter: blur(10px) brightness(0.24) saturate(0);
+    box-shadow: inset 0 0 0 $column__border-width var(--accent-color--dark);
+    background: rgba(var(--accent-color-rgb), 0.1);
+  }
+}
```

Comment:
why move this up?

Reply:
Because personally I prefer it that the dark theming section for a specific classname is right underneath the normal styling block